src/component/Button.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+function Button(props) {
+  const { buttonName, color, wide } = props;
+  const buttonColor = color === "grey" ? "grey" : "orange"
```

Comment:
we are supposed to use defaultProps for colour. so any button that doesn't get a prop will automatically use orange.